### PR TITLE
[Migration] Updates to 6953474 snapshot

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: MigrationCodelab/gradle.properties [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-android.enableJetifier=true
+android.enableJetifier=false
```
Comment (chrisbanes) [inline MigrationCodelab/gradle.properties:28]: Can remove this line (it's the default)

Comment (manuelvicnt) [reply MigrationCodelab/gradle.properties]: Done, thanks!

**Review by chrisbanes — APPROVED** [review]
LGTM, just one minor comment

File: MigrationCodelab/build.gradle [hunk 2]
```diff
@@ -68,8 +68,23 @@ plugins {
...
+        maven { // Compose alpha07 snapshot
+            url "https://androidx.dev/snapshots/builds/6953474/artifacts/repository/"
+        }
         google()
         jcenter()
+        maven { url 'https://jitpack.io' } // Needed for Compose's MDC Theme adapter
+    }
+}
+
+subprojects {
```
[the remark this answers is not in the document]
Comment (manuelvicnt) [reply MigrationCodelab/build.gradle:80]: Sure, this should go away when updating to `alpha07` cc @florina-muntenescu

**Pull request merged**